### PR TITLE
Domain Transfer: Screen 2 fixes/changes

### PR DESCRIPTION
**Pull request by renancarvalho**
## Proposed Changes

Fixes comments from screen 2 in this thread p9Jlb4-8kT-p2

## Testing Instructions

- Pull and run this branch or use calypso live link
- Navigate to setup/domain-transfer/domains
- Check if the comments are fixed

Some comments are not ready to be fixed, they still being discussed

- Field labels are misaligned.
- Cacophony of Add domains.
- The disabled stuff is awful.
- Gridicon (refresh) vs G2 icon (trash).

### CODE REVIEW

**Review by escapemanuele — APPROVED** [review]
Tested and it works great! Should we wait for confirmation on the missing things or just merge?

**Comment by renancarvalho** [comment]
Hi @escapemanuele, thank you for checking it out. I am updating small changes and we should be good to go.

**Review by lupus2k — APPROVED** [review]
LGTM, tested and all shows as expected!

File: client/landing/stepper/declarative-flow/internals/steps-repository/domain-transfer-domains/domain-code-pair.tsx [hunk 1]
```diff
@@ -60,7 +62,11 @@ const DomainPrice = ( { rawPrice, saleCost, currencyCode = 'USD' }: DomainPriceP
 	const { __ } = useI18n();
 
 	if ( ! rawPrice ) {
-		return <div className="domains__domain-price-number disabled">0</div>;
+		return (
+			<div className="domains__domain-price-number disabled">
```
Comment (ebinnion) [inline client/landing/stepper/declarative-flow/internals/steps-repository/domain-transfer-domains/domain-code-pair.tsx:66]: Testing locally, I can confirm that I'm getting more correct price formatting. It still always seems to be in USD though, even if I sign up with a new user. Perhaps just an issue with how I'm testing.

Screenshot 2023-07-19 at 2 59 31 PM

**Comment by ebinnion** [comment]
I rebased against trunk and pushed one commit to soften the subheading a bit. Will wait for tests to pass and then merge.

cc @nuriapenya as we may want to revisit the copy on this subheading in your morning.

**Comment by allilevine** [comment]
The field labels need to be aligned on the x-axis:

Screen Shot 2023-07-19 at 4 51 39 PM

From p9Jlb4-8kT-p2#comment-8645

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/8600734

Hi @renancarvalho, could you please edit the description of this PR and add a screenshot for our translators? Ideally it'd include this string: `Enter your domain names and authorization codes below. You can transfer up to 50 domains at a time.`

Thank you in advance!

File: client/landing/stepper/declarative-flow/internals/steps-repository/domain-transfer-domains/domain-code-pair.tsx (hunk 1 above)
Comment (lsl) [reply client/landing/stepper/declarative-flow/internals/steps-repository/domain-transfer-domains/domain-code-pair.tsx:66]: Currency gets geolocated for logged out users regardless of locale. "33 US$" is the kind of thing you see when you have a french locale but currency set to USD.

Logged in, you can change your user's currency via SA, that's usually how I test different currencies, to test the currency detection itself you need a vpn.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/8600734).

**Pull request merged**